src/services/catalog.service.ts

```diff
@@ -22,26 +22,42 @@ export class CatalogService {
   }
 
   public getCatalogItems(includeTemplates: boolean) {
-    let promises: any = {
-      serviceClasses: this.dataService.list({
-        group: 'servicecatalog.k8s.io',
-        resource: 'serviceclasses'
-      }, {}),
-      imageStreams: this.dataService.list("imagestreams", { namespace: "openshift" })
-    };
+    let deferred = this.$q.defer();
+    let catalogItems: any = {};
+    let totalNumPromises: number = includeTemplates ? 3 : 2;
+    let numPromisesExecuted: number = 0;
+    let errorMsg: string = '';
+
+    this.dataService.list({
+      group: 'servicecatalog.k8s.io',
+      resource: 'serviceclasses'
+    }, {}).then( (resources: any) => {
+      catalogItems.serviceClasses = resources.by("metadata.name");
+    }, () => {
+      errorMsg += 'Error loading Service Classes.';
```

Comment:
as a rule in the console we lower case types when they are part of a sentence.  "Error loading service classes."